### ISSUE TITLE
No Advancements

### ISSUE THREAD

**Issue by Goobercheese** [BSData/bloodbowl#57]
Ever since downloading the latest data, I am no longer able to advance my players on any sort of list. Tap the player, give them enough SPP, no advancements option appears. Open my old lists, all advancements are gone and again no option to add them in.

**Comment by shaner11** [comment]
I haven't been able to recreate this myself, but have seen a couple other comments mentioning it. Could you provide further details/screenshots to help me investigate?

**Comment by Goobercheese** [comment]
Sure, I only use the mobile version personally, but I hope these help. 
![Screenshot_20220602-175534](https://user-images.githubusercontent.com/106558347/171765707-70cb52a1-968d-40c2-8fb5-cae958744467.png)
And here's what happens when I try to open an old roster with leveled up players: 
![Screenshot_20220602-175913](https://user-images.githubusercontent.com/106558347/171766373-3adf0cb5-46d8-47f9-b1fe-5970bbdefb6f.png)

**Comment by Goobercheese** [comment]
I didn't realize the screencaps would be so big, sorry!

**Comment by shaner11** [comment]
I've just put out an update that should solve this one. Apologies for the wait and a massive thank you for everyone who reported this and especially to those who gave me further details to help track down the issue.

tl;dr: I forgot to change a number and it broke things. Apologies for the stupid error.

It turns out the issue was an unchanged version number in the game system file. For the unaware, when Battlescribe updates a game system it checks the version numbers of each game system and faction in that system, and only updates files with a lower version number than the updated files. This result here was that the new advancements entry wasn't added when the files were updated, so they couldn't be added. @dumdrak mentioned that doing a fresh download of all files fixed the issue, which is what tipped me off. I was testing with fresh downloads as well, which is why I wasn't able to recreate the issue. It was likely this was causing other strange issues (inconsistent rule text, missing star players/inducements, incorrect pricing on some options, etc.) that this should also solve.

**Comment by Goobercheese** [comment]
Thanks so much!
